apps/api/src/app/order/order.service.ts

```diff
@@ -541,4 +545,21 @@ export class OrderService {
       where
     });
   }
+
+  public async getCountAndDateMin(
```

Comment:
Maybe `getStatisticsByCurrency`❓

Reply:
Definitely better. I can also suggest adding something that indicates where these stats are relevant, e.g., `getActivityStatsByCurrency`.